soc/arm64/renesas_rcar/gen3/mmu_regions.c

```diff
@@ -21,13 +21,6 @@ static const struct arm_mmu_region mmu_regions[] = {
 			      DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 1),
 			      MT_DEVICE_nGnRnE | MT_RW | MT_NS),
 
-#if DT_NODE_HAS_STATUS(DT_INST(0, renesas_rcar_scif), okay)
```

Comment:
Shouldn't you fold this change into "drivers: uart_rcar: do runtime MMIO mapping"?

I am asking, because your previous patch can break Zephyr is applied without this change here.

Reply:
ok, I'll squash commits together